docs/decisions/0017-reimplement-asset-processing.rst

```diff
@@ -128,9 +130,11 @@ The three top-level edx-platform asset processing actions are *build*, *collect*
 
        A Python-defined task that calls out to each build stage.
 
-     - ``scripts/build-assets.sh``
+     - ``npm clean-install && npm run build``
+
+       Simple NPM wrappers around the build stages. The wrappers will be written in Bash and tested on both GNU+Linux and macOS.
 
-       A Bash script that contains all build stages, with subcommands available for running each stage separately. Its command-line interface inspired by Tutor's ``openedx-assets`` script. The script will be runnable on any POSIX system, including macOS and Ubuntu and it will linted for common shell scripting mistakes using `shellcheck <https://www.shellcheck.net>`_.
+       These commands are a "one stop shop" for building assets, but more efficiency-oriented users may choose to run build stages individually.
```

Comment:
As I moved towards ``npm run``, I moved away from the structure of ``openedx-assets``.